frontend/server/k8s-helper.ts

```diff
@@ -127,3 +131,30 @@ export function getPodLogs(podName: string): Promise<string> {
       (error: any) => { throw new Error(JSON.stringify(error.body)); }
     );
 }
+
+export async function newTensorboardInstance(logdir: string): Promise<void> {
+  if (!k8sV1CustomObjectClient) {
+    throw new Error('Cannot access kubernetes Custom Object API');
+  }
+  // TODO: take the configuration below to a separate file
+  const group = 'kubeflow.org';
+  const version = 'v1beta1';
+  const namespace = 'kubeflow';
+  const plural = 'tensorboard';
+  const body = {
+    apiVersion: 'kubeflow.org/v1beta1',
+    kind: 'Viewer',
+    metadata: {
+      generateName: 'viewer-',
+      namespace: 'kubeflow',
+    },
+    spec: {
+      type: 'tensorboard',
+      tensorboardSpec: {
+        logDir: logdir,
+      }
+    }
+  };
+  const pretty = null;
```

Comment:
`pretty` is an optional parameter to the `createNamespacedCustomObject`, so you can just ignore it and call it with
`createNamespacedCustomObject(group, version, namespace, plural, body)`

Reply:
Done